kernel/src/main/scala/me/rexim/morganey/church/ChurchPairConverter.scala

```diff
@@ -16,11 +16,23 @@ object ChurchPairConverter {
       case _ => None
     }
 
-  def encodePair(p: (LambdaTerm, LambdaTerm), cons: String = "x"): LambdaTerm = {
-    val (car, cdr) = p
-    LambdaFunc(LambdaVar(cons),
-      LambdaApp(LambdaApp(LambdaVar(cons), car), cdr)
-    )
+  def encodePair(pair: (LambdaTerm, LambdaTerm)): LambdaTerm = {
+    val (first, second) = pair
+    val x = LambdaVar("x")
+
+    val pairFunc = {
+      val y = LambdaVar("y")
+      val z = LambdaVar("z")
+
+      LambdaFunc(y,
+        LambdaFunc(z,
+          LambdaApp(LambdaApp(z, x), y)))
```

Comment:
Look, beforehand I could do:
```
λ> :raw [1,2,3]
λx.x (λf.x.f x) (λx.x (λf.x.f (f x)) (λx.x (λf.x.f (f (f x))) (λf.x.x)))
```

Now I have:
```
λ> :raw [1,2,3]
λz.z (λf.x.f x) (λz.z (λf.x.f (f x)) (λz.z (λf.x.f (f (f x))) (λf.x.x)))
```
I don't like that `z` comes first now.

Could we rearrange the variables here to make a prettier pattern? Like `λx.x (λf.y.f y) ...`?